CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # CHANGELOG
+## 3.5.0
+#### New Functionality
+
+- Can now use BM25 and vector scoring method. (MarkLogic Server version 12.0.0-EA or higher needed).
```

Comment:
Make two separate items here because they're two separate features:

1. Can use BM25 scoring method for searches. 
2. Can use new vector functions in Optic queries.